cpu/cortexm_common/ldscripts/cortexm_base.ld

```diff
@@ -42,8 +42,16 @@ SECTIONS
     .text :
     {
         . = ALIGN(4);
-        _sfixed = .;
-        KEEP(*(.vectors .vectors.*))
+        _stack_base = .;
+        LONG(ABSOLUTE(_estack))
```

Comment:
while this is still a valid way of doing this I think that the readability of the code is better if the whole common part of the ISR vector table is inside `cpu/cortexm_common/vectors_cortexm.c` instead.